### PR TITLE
Makes the camera function optional

### PR DESCRIPTION
**Pull request by AxelHamburch**
This PR expands the config.ini by asking whether a camera is available and by asking which language you want to use. Only the part with the camera is active here. The language part follows in a subsequent PR. 

The function is that you can specify whether a camera is available or not. It is disabled by default as most will have the pocket version. Now that we know that there is no camera, we can also skip or disable functions that only make sense when the camera is active. For example, the page with the query whether LNURL is possible. If not, then the camera has been activated. This site is useless for people without a camera and without a button. So we can skip the function right now and people don't have to laboriously edit this page call in the app.py code anymore.

The next point is the function that is called when the button is pressed 5 times. All wallet data is currently being deleted and there is the option of rescanning the data. But if I don't have a camera, only my data is erased and I have to laboriously re-enter it by hand. That's why we can block this function in the future if you don't have a camera.

### CODE REVIEW

**Review by AxelHamburch — COMMENTED** [review]
Good for me. Tested it several times.

File: app.py [hunk 1]
```diff
@@ -111,12 +111,13 @@ def button_pushed():
...
         activewallet = config.conf["atm"]["activewallet"]
+        camera = config.conf["atm"]["camera"]
         # Determine if LNURL Withdrawls are possible
         if lnurlproxy == "active" or activewallet == "lntxbot":
-            # 1. Ask if wallet supports LNURL
+            # 1. Ask if wallet supports LNURL, if camera available
             # 2. Offer to cancel and switch to normal scan
             # 3. Process payment
-            if activewallet == "lntxbot":
+            if activewallet == "lntxbot" and camera == "yes":
```
Comment (lightisfaster) [inline app.py]: boolean true or false is more common

**Review by lightisfaster — APPROVED** [review]
Thanks again for this PR!
Boolean true or false is more common for switch on or off a var (camera).

**Comment by lightisfaster** [comment]
I've just made a small change for the boolean flag camera.

**Pull request merged**